test/v1/fixtures/ListBenefitsFixture.scala

```diff
@@ -901,6 +901,244 @@ object ListBenefitsFixture {
       |}
       |""".stripMargin)
 
+  val mtdTysJson: JsValue = Json.parse(s"""
```

Comment:
rather than adding mtdTysJson just to change the taxYear in the links cant we turn mtdJson into a function that takes taxYear: String as a parameter and replace the year in the hateoas lnks with $taxYear?